### ISSUE TITLE
R code

### ISSUE THREAD

**Issue by semperparatusalpha** [viscom-ulm/Net2Vis#6]
Greetings,

Could the same visualization be run with R code?

Take care

**Comment by Sparkier** [comment]
So far this is not possible as the application was designed to work with Keras models. The technique could, however, be extended to other platforms as well, I just currently do not have the time to do so.

If you would like to create a PR for R based models, here are some pointers:

To render the visualization, I get a model graph from the backend (https://github.com/viscom-ulm/Net2Vis/blob/305979027ef4a4eb1268e90012aaa7ef19e52b15/backend/server.py#L152-L153).
This model graph has the following definition: https://github.com/viscom-ulm/Net2Vis/blob/master/backend/translate/graph.py and consists of multiple layers (https://github.com/viscom-ulm/Net2Vis/blob/master/backend/translate/layer.py), which the frontend is able to render.

Basically what you would need to do to make this possible is write a translate function, such as this one for Keras (https://github.com/viscom-ulm/Net2Vis/blob/305979027ef4a4eb1268e90012aaa7ef19e52b15/backend/translate/translate_keras.py#L9-L28) to translate your network into my Graph representation.

If you have any more questions feel free to ask.

**Comment by Sparkier** [comment]
Closing this as there has been not activity for a while, and I dont plan to integrate R support myself.
